Staging-Updates-2.0/u2-sce-test-wso2ob-2.0.0-AU-staging-testgrid.yaml

```diff
@@ -0,0 +1,46 @@
+# TestGrid Test Configuration Parameters
+version: '0.9'
+emailToList: "bfsi-group@wso2.com,divya@wso2.com,malshani@wso2.com,verushka@wso2.com,kasundh@wso2.com,charithad@wso2.com,akilaa@wso2.com,rivindu@wso2.com,thiloshon@wso2.com"
```

Comment:
Email list needs to be updated. Right ?

Reply:
You mean to the following values or else can you suggest?
----------------------------
maheshika@wso2.com,
shariq@wso2.com,
vimukthi@wso2.com,
integration-builder@wso2.com,
iam-builder@wso2.com,
bfsi-group@wso2.com,
ashen@wso2.com,
chanika@wso2.com,
dilini@wso2.com,
divya@wso2.com,
jayangak@wso2.com,
malshani@wso2.com,
mushthaq@wso2.com,
prasanna@wso2.com,
paraparan@wso2.com
-------------------------------

Reply:
@AmilaSamith , I think what @Dinushi  means is that the mail "bfsi-group@wso2.com" should be changed to "integration-bfsi-group@wso2.com".

Reply:
Thanks for the clarification.I'll make the changes.

Reply:
Removed outdated recipients form email list